pkgs/applications/misc/limesuite/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, libusb, cmake, wxGTK30, mesa, libX11, soapysdr, sqlite, doxygen, gnuplot ? null }:
+
+stdenv.mkDerivation rec {
+  version = "17.12.0";
+  name = "LimeSuite-${version}";
+
+  src = fetchurl {
```

Comment:
I think `fetchFromGitHub` is the preferred method:
  src = fetchFromGitHub {
    owner = "myriadrf";
    repo = "LimeSuite";
    rev = "v${version}";
    sha256 = "1rd79xxnwr62fmy2vq895jyiqnz6lm4p8bl09adjqqm4y5npryf9";
  };